internal/test/wsu/wsu_test.go

```diff
@@ -224,20 +256,6 @@ func testFilesCopied(t *testing.T, vm e2ef.WindowsVM, ansibleTempDir string) {
 		assert.NoError(t, err, "Error looking for %s: %s", fullPath, err)
 		assert.Emptyf(t, stdout, "Missing file: %s", fullPath)
 	}
-
-	// Check the SHA of kube.tar.gz downloaded
-	kubeTarPath := ansibleTempDir + "\\" + "kube.tar.gz"
-	// certutil is part of default OS installation Windows 7+
-	command := fmt.Sprintf("certutil -hashfile %s SHA512", kubeTarPath)
-	stdout, stderr, err := vm.Run(command, false)
-	require.NoError(t, err, "Error generating SHA512 for %s", kubeTarPath)
-	require.Equalf(t, "", stderr, "Error generating SHA512 for %s", kubeTarPath)
-	// CertUtil output example:
-	// SHA512 hash of <filepath>:\r\n<SHA-output>\r\nCertUtil: -hashfile command completed successfully.
-	// Extracting SHA value from the output
-	actualKubeTarSha := strings.Split(stdout, "\r\n")[1]
-	assert.Equal(t, expectedKubeTarSha, actualKubeTarSha,
-		"kube.tar.gz downloaded does not match expected checksum")
```

Comment:
Why Is this block being removed?

Reply:
Since we are downloading kube node binaries dynamically now, I moved the SHA check to WSU playbook